services/iota_wrappers.go

```diff
@@ -41,8 +44,7 @@ type IotaService struct {
 	FindTransactions
 	GetTransactionsToApprove
 }
-
-type ProcessingFrequency struct {
+type ProcessingFhttpRequency struct {
```

Comment:
Is `ProcessingFhttpRequency` a spelling error?

Reply:
yeah, didnt mean to change that. Good catch